src/wp-includes/class-wp-theme-json.php

```diff
@@ -2105,6 +2121,12 @@ protected static function compute_style_properties( $styles, $settings = array()
 				}
 			}
 
+			// Processes background styles.
+			if ( 'background' === $value_path[0] && isset( $styles['background'] ) ) {
+				$background_styles = wp_style_engine_get_styles( array( 'background' => $styles['background'] ) );
+				$value             = $background_styles['declarations'][ $css_property ] ?? $value;
```

Comment:
Should this `??` be an `isset`?

Reply:
Good catch. That one slipped through the net. It's not a very good net. 😄